service-base/src/main/java/org/eclipse/hono/service/http/AbstractHttpEndpoint.java

```diff
@@ -395,4 +397,52 @@ protected final CorsHandler createCorsHandler(final String allowedOrigin, final
                 .allowedHeader(HttpHeaders.IF_MATCH.toString())
                 .exposedHeader(HttpHeaders.ETAG.toString());
     }
+
+    /**
+     * Creates a response based on generic result.
+     * <p>
+     * The default behavior is as follows:
```

Comment:
this sounds like the *default* behavior might be changed somehow, but since this method is final, I guess that this is *the behavior*, isn't it?